PlaybackLab/DownloadToOwnSampleApp/README.md

```diff
@@ -20,21 +20,19 @@ To run this app, we assume you have an Apple developer account because you need
 ## App Architecture
 
 The app uses the following classes provided by the _OoyalaSDK_:
-* **OOAssetDownloadManager**: Downloads a single video asset. Needs an `OOAssetDownloadOptions` to initialize.
-* **OOAssetDownloadOptions**: Provides information to `OOAssetDownloadManager` so it knows which asset to download.
-* **OOEmbedTokenGenerator**: An instance of this protocol is used to enable our Ooyala Player Token (OPT) protection feature. This is necessary for assets using entitlements and Fairplay DRM protected assets. When downloading a Fairplay asset, you must provide an `OOEmbedTokenGenerator` instance to `OOAssetDownloadOptions` before creating an `OOAssetDownloadManager`.
-* **OOAssetDownloadManagerDelegate**: Build a class implementing this protocol to know about the progress of a download. Remember setting the class as a delegate of `OOAssetDownloadManager`.
+* **OODtoAsset**: Downloads a single video asset. Needs an `PlayerSelectionOption` to initialize. Use appropriate methods with closures `-downloadWithProgressClosure: -progressWithProgressClosure: -finishWithRelativePath: -onErrorWithErrorClosure:` to start downloading and to know about the progress of a download.
+* **PlayerSelectionOption**: Provides information to `OOAssetDownloadOptions` so it knows which asset to download.
```

Comment:
Remove this line since it will confuse customers.
`PlayerSelectionOptions` is just a helper class